crates/core/src/db/relational_db.rs

```diff
@@ -283,13 +283,17 @@ impl RelationalDB {
     #[tracing::instrument(skip_all)]
     pub fn begin_mut_tx(&self, isolation_level: IsolationLevel) -> MutTx {
         log::trace!("BEGIN MUT TX");
-        self.inner.begin_mut_tx(isolation_level)
+        let r = self.inner.begin_mut_tx(isolation_level);
+        log::trace!("ACQUIRED MUT TX");
+        r
     }
 
     #[tracing::instrument(skip_all)]
     pub fn begin_tx(&self) -> Tx {
         log::trace!("BEGIN TX");
-        self.inner.begin_tx()
+        let r = self.inner.begin_tx();
+        log::trace!("ACQUIRED TX");
```

Comment:
when we were debugging lock issues, we appreciated not only having a "trying to acquire lock" message but also a "I am now holding the lock" message.